### ISSUE TITLE
Req: would be interface bonding intrested?

### ISSUE THREAD

**Issue by amdei** [nMustaki/debinterface#33]
Hello,
I'm planning to add support for configuring network interfaces bonding via ifenslave.
How do you think, would it be interested?
Will you accept PR with such functionality?

**Comment by nMustaki** [comment]
Hello @amdei,

I don't know ifenslave would you mind giving some pointers ?
I would be glad to add functionalities to the library !

Thanks for contributing :-)

**Comment by ObjatieGroba** [comment]
Hello, @nMustaki 

I would try to do it. There is API draft
Can you check API and make the adjustments?

Thank you

**Comment by amdei** [comment]
>  would you mind giving some pointers ?

Sure, here are some articles to start: 
https://help.ubuntu.com/community/UbuntuBonding
https://www.tecmint.com/network-nic-bonding-teaming-in-debian-linux/

**TL;DR:**
```
apt-get install ifenslave
echo 'bonding' >> /etc/modules 

/etc/network/interfaces
auto bond0
iface bond0 inet static
    address 192.168.1.10
    gateway 192.168.1.1
    netmask 255.255.255.0

    bond-mode 4
    bond-miimon 100
    bond-lacp-rate 1
    bond-slaves eth0 eth1
    up /sbin/ifenslave bond0 eth0 eth1
    down /sbin/ifenslave -d bond0 eth0 eth1

```

**Comment by nMustaki** [comment]
Hello,

I've been completely overwhelmed by work for the past months...
I promise to give you a real response by the end of the week

Thanks you for putting up with me :-)


--
/*
** "What do you despise? By this you are truly known."
**	from Manual of Muad'Dib by the Princess Irulan.
*/

**Comment by ObjatieGroba** [comment]
Hi, @nMustaki 

Can you review PR?

Thank you.
